apps/schools/domains/circles/redux/interfaces.ts

```diff
@@ -0,0 +1,40 @@
+interface ReturnedData<T> {
+    count: number,
+    next: string,
+    previous: string,
+    results: T,
+}
```

Comment:
it is common pagination interface
we should locate it to common interfaces and rename to PaginationResponse